docs/_layouts/default.html

```diff
@@ -84,6 +84,7 @@ <h1 class="title">
         </div>
 
         <!-- Concierge button at the bottom of the page -->
+        {% include CONST.html %}
```

Comment:
While this would work seamlessly for `development` environment, we also need to confirm if the main site url would get set for `production` environment as mentioned within CONST.html. Can you please investigate this in code to confirm this?